### PR TITLE
Add support for the Push Notifications Devices destroy_many call.

### PR DESCRIPTION
**Pull request by andreionut**
/cc @ggrossman @shanitang 

This is for an upcoming API call: https://developer.zendesk.com/rest_api/docs/core/push_notification_devices

### CODE REVIEW

**Comment by shanitang** [comment]
@andreionut Thanks for this! Could you add a unit test as well?

**Comment by andreionut** [comment]
bump

File: src/Zendesk/API/PushNotificationDevices.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    /**
+     * @param Client $client
+     */
+    public function __construct(Client $client) {
+        parent::__construct($client);
+    }
+
+    /**
+     * Delete a ticket or series of tickets
```
Comment (shanitang) [inline src/Zendesk/API/PushNotificationDevices.php]: This should be "Unregister the mobile devices that are receiving push notifications" ?

Comment (andreionut) [inline src/Zendesk/API/PushNotificationDevices.php]: Yes, thanks for pointing this out.

**Comment by andreionut** [comment]
We went live with the API endpoint today. If there aren't any objections can I get a +1 so I can merge?

**Comment by shanitang** [comment]
:+1:

**Pull request merged**